shared-bindings/socketpool/SocketPool.c

```diff
@@ -52,7 +52,6 @@
 //|             returned by :py:attr:`wifi.radio`
 //|         """
 //|         ...
-//|
```

Comment:
hmm, weird, on my local machine, pre-commit Formatting make this change and failed otherwise. But ci pre-commit seems to decide the other way around. Probably due to version difference, I have to temporarily uninstall pre-commit to force revert this change.